services-js/commissions-search/src/App.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+// import './App.css';
+import update from "immutability-helper";
+
+import Search from "./Search";
+import FacetList from "./FacetList";
+import ResultList from "./ResultList";
+
+class App extends Component {
+  state = {
+    tempKeywords: "",
+    keywords: "",
+    tempFacets: {},
+    facets: {},
+    tempSeats: "null",
```

Comment:
It would be good to have an explanatory comment for why this is a string literal `"null"` instead of just the value `null`.

Reply:
Also, just put a comment in about `temp` vs. non-`temp` because the function (current form state vs. submitted form state) is not obvious from the names.

(You also could try to improve the names as well to make it more obvious what each is for.)